testkit/src/main/scala/org/bitcoins/testkit/rpc/BitcoindRpcTestUtil.scala

```diff
@@ -223,12 +226,14 @@ trait BitcoindRpcTestUtil extends BitcoinSLogger {
       versionOpt: Option[BitcoindVersion] = None): BitcoindInstance = {
     val uri = new URI("http://localhost:" + port)
     val rpcUri = new URI("http://localhost:" + rpcPort)
-    val configFile = writtenConfig(
-      uri,
-      rpcUri,
-      zmqPort,
-      pruneMode,
-      blockFilterIndex = versionOpt.contains(BitcoindVersion.Experimental))
+    val hasNeutrinoSupport = versionOpt.contains(BitcoindVersion.V19) || versionOpt
```

Comment:
is this right? `v19` doesn't have p2p neutrino support does it? What type of neutrino support are we talking about? We should probably be more specific if we are talking about cli or p2p

Reply:
This is correct, this means that the `blockfilterindex` config option works on the version of bitcoind